frontend/src/components/CloseSuspendReasonModal.css

```diff
@@ -0,0 +1,30 @@
+.smart-hub--goal-close-suspend-reason .usa-form-group fieldset textarea {
+    resize: none;
+    width: 415px;
+}
+
+.usa-modal__footer .usa-button-group__item .usa-button--unstyled {
+    padding-left: 8px;
+    padding-right: 8px;
+    margin-right: 8px;
+    margin-left: 8px;
+}
+
+
+.popup-modal .usa-modal__main h2 {
+    margin-bottom: 0px;
```

Comment:
This and some of the other CSS in this file apply to all modals, should it go in a different file, maybe Modal.css?

Reply:
Its not for all modals just this one.

Reply:
Added close/suspend modal prefix in css.